core/Cargo.toml

```diff
@@ -88,6 +88,8 @@ layers-madsim = ["dep:madsim"]
 layers-minitrace = ["dep:minitrace"]
 # Enable layers tracing support.
 layers-tracing = ["dep:tracing"]
+# Enable layers oteltrace support.
+layers-oteltrace = ["dep:opentelemetry"]
```

Comment:
I expect to use `layers-otel-trace` to reflect it's project structure. In the future, we may have `layers-otel-metrics`.

Reply:
fixed